go/cmd/vtorc/index.go

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright 2022 The Vitess Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"net/http"
+)
+
+// This is a separate file, so it can be selectively included/excluded from
+// builds to opt in/out of the redirect.
```

Comment:
does anyone _actually_ leverage this "feature"? deleting files and rebuilding vitess from scratch seems like a kind of heavy-handed way to avoid a redirect, so i'm not actually sure how much we've benefitted from this (i know this is done in other places but i've always been skeptical)

Reply:
Me too. I didn't understand it, so copied it over. Maybe @deepthi knows why we do it this way 🤷‍♂️

Reply:
I have no idea :(

Reply:
I removed that comment since it doesn't make any sense, at least not that any of us can make any.